orttraining/orttraining/lazy_tensor/bridge.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <ATen/core/ivalue.h>
+#include "core/framework/ortdevice.h"
+#include "core/framework/ort_value.h"
+
+namespace onnxruntime {
+namespace lazytensor {
+// Scalar type translation from ONNX to Pytorch.
+c10::ScalarType CreateC10ScalarType(const onnxruntime::PrimitiveDataTypeBase* elem_type);
+// Scalar type translation from Pytorch to ORT.
+onnxruntime::MLDataType CreateOrtScalarType(at::ScalarType dtype);
+// Device translation from Pytorch to ORT.
+OrtDevice CreateOrtDevice(const c10::Device device);
```

Comment:
one headache, we have an "ort" virtual device in pytorch, and we will rely on it for our 1PNPU workload. let's see how to handle it in the future.

Reply:
This `bridge` code is for creating ORT tensor from Aten tensor. If JORT sees device `ort`, then it can just cast it to `OrtAtenTensor` without copying data.